packages/backend/src/utils/JwtUtil.ts

```diff
@@ -0,0 +1,100 @@
+import {
+    CreateEmbedJwt,
+    EmbedJwt,
+    EmbedJwtSchema,
+    ForbiddenError,
+    ParameterError,
+    getErrorMessage,
+    isDashboardUuidContent,
+} from '@lightdash/common';
+import * as Sentry from '@sentry/node';
+import jwt from 'jsonwebtoken';
+import { z } from 'zod';
+import { lightdashConfig } from '../config/lightdashConfig';
+import Logger from '../logging/logger';
+import { EncryptionUtil } from './EncryptionUtil/EncryptionUtil';
+
+export const JWT_HEADER_NAME = 'lightdash-embed-token';
+
+/**
+ * Encodes JWT data into a token
+ */
+export function encodeJwt(
```

Comment:
only suggestion now these are outside a service is to give them a name that suggests they check the jwt schema that's lightdash specific - they aren't a generic jwt

e.g. `decodeLightdashJWT`

Reply:
That's true. I'll update

Reply:
Updated the names and moved to a separate auth package in backend 🚀